kyuubi-server/src/main/scala/org/apache/kyuubi/operation/ExecuteStatement.scala

```diff
@@ -124,6 +124,11 @@ class ExecuteStatement(
           case CANCELED_STATE =>
             setState(OperationState.CANCELED)
 
+          case TIMEDOUT_STATE
+              if getSession.protocol.getValue <=
+                TProtocolVersion.HIVE_CLI_SERVICE_PROTOCOL_V8.getValue =>
```

Comment:
can we also add some comments to describe the reason why we do this

Reply:
done